LondonBritishLibrary/orient/BLorient12513.xml

```diff
@@ -0,0 +1,658 @@
+                                    </support>
+                                    <extent>
+                                        <measure unit="leaf">1+147+1</measure>
+                                        <measure unit="page" type="blank">8</measure><locus target="#Iv #4r #4v #5r #5v #10r #147v #IIv"/>
+                                        <dimensions type="outer" unit="mm">
+                                            <height>285</height>
+                                            <width>245</width>
+                                        </dimensions>
+                                    </extent>
+                                </supportDesc>
+                            </objectDesc>   
+                            <additions>
+                                <list>
+                                    <item xml:id="a1">
+                                        <locus target="#10v"/>
+                                        <desc type="PurchaseNote">Amharic note concerning the purchase of this manuscript for five thalers by a certain
+                                            <persName ref="PRS14488Afawarq">ʾAfawarq</persName></desc>
+                                        <q xml:lang="am">ጌታ፡ ባ፭ብር፡ የገዙት፡ <persName ref="PRS14488Afawarq">አፈወርቅ፡</persName> ነው፡</q>
+                                    </item>
+                                    <item xml:id="e1">
+                                        <locus target="#Ir"/>
+                                        <desc type="MixedNote">A note in pencil</desc>
+                                        <q xml:lang="en">The Acts of the Apostles<sic/> in the Ethiopic Language from King Theodore's Palace</q>
+                                    </item>
+                                    <item xml:id="e2">
+                                        <locus target="#IIr"/>
+                                        <desc type="MixedNote">Pen trial</desc>
+                                    </item>
+                                </list>
+                            </additions>
+                            <bindingDesc>
+                                <binding xml:id="binding">
+                                    <decoNote xml:id="b1" type="Boards"><material key="leather"/>Wooden boards, repaired in the 
+                                        <placeName ref="INS00001BL">British Museum</placeName>.</decoNote>
+                                    <decoNote xml:id="b2" type="Cover"><material key="leather"/>Partly covered with leather.</decoNote>
+                                </binding>
+                            </bindingDesc>
+                        </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1600" notAfter="1699">17th century</origDate>    
+                        </origin>
+                        <acquisition>Presented by the <placeName ref="INS0775LWC">Wellcome Institute</placeName>, 
```

Comment:
If you present a manuscript like a composite one, you need to articulate the reasons why you did that (esp. if the source description does not do this). Indeed, what are the reasons? The description of Strelcyn does not contain any hint to that, as far as I can see.

Reply:
I assume, that his dating is based on palaeography. I think that, because there is no other reason indicated by Strelcyn.